tutorials/getting_started/rules_advanced.md

```diff
@@ -76,26 +94,51 @@ We will use the same trigger from the Blockly tutorial.
 
 ### Then: Actions
 
-Just like in Blockly, we will create a new Action but instead of choosing Blockly, we will choose ECMAScript 2021.
+Just like in Blockly, we will create a new Action but instead of choosing Blockly, we will choose `ECMAScript` or `Ruby`.
 
 ![select language](images/rules-advanced-select-lang.png)
 
 This will open a blank text field where you can start typing your code.
 This text field provides text highlighting and some code completion which helps with coding.
 
+As with the Blockly example, we want to start the rule using a log statement we can see in openhab.log when the rule runs.
+
+:::: tabs
+
+::: tab JS
+
 As previously mentioned, the Helper Library for this language comes with the add-on and is imported by default, so see the [JavaScript Scripting add-on's reference](/addons/automation/jsscripting/) for the full guide on how to do anything you might want to do.
 For help with general JavaScript coding, there are tons of tutorials and reference documents on the web a search away.
 
-As with the Blockly example, we want to start the rule using a log statement we can see in openhab.log when the rule runs.
 One can either use the [log actions](/addons/automation/jsscripting/#log) but most will find it easiest to use the more JavaScript native [`console`](/addons/automation/jsscripting/#console).
 
 ```javascript
 console.info('Motion was detected');
 ```
 
+:::
+
+::: tab JRuby
+
+If you are new to Ruby, check out [Ruby Basics](https://openhab.github.io/openhab-jruby/main/file.ruby-basics.html) for a quick overview of the language.
+The Ruby language and the [JRuby Helper Library](https://openhab.github.io/openhab-jruby/) offers a streamlined syntax for writing file-based and UI-based rules, making it easier and more intuitive than RulesDSL, while delivering the full features of the Ruby language.
```

Comment:
```suggestion
The Ruby language and the [JRuby Helper Library](https://openhab.github.io/openhab-jruby/) offers a streamlined syntax for writing file-based and UI-based rules, making it easy and intuitive to work with openHAB constructs, while also delivering the full features of the Ruby language.
```

I don't think we need to specifically compare to other languages, since that's subjective.

Reply:
This is from an old text I've been copy pasting into our usage / readme etc. I'll update them too.